server/graphql/v2/object/TransactionsImport.ts

```diff
@@ -86,15 +89,54 @@ export const GraphQLTransactionsImport = new GraphQLObjectType({
     rows: {
       type: new GraphQLNonNull(GraphQLTransactionsImportRowCollection),
       description: 'List of rows in the import',
-      resolve: async importInstance => {
-        const where = { TransactionsImportId: importInstance.id };
+      args: {
+        ...getCollectionArgs({ limit: 100 }),
+        status: {
+          type: GraphQLTransactionsImportRowStatus,
+          description: 'Filter rows by status',
+        },
+        searchTerm: {
+          type: GraphQLString,
+          description: 'Search by text',
+        },
+      },
+      resolve: async (
+        importInstance,
+        args: { limit: number; offset: number; status: TransactionsImportRowStatus; searchTerm: string },
+      ) => {
+        const where: Parameters<typeof TransactionsImportRow.findAll>[0]['where'] = {
```

Comment:
Something to work on some other time: make a generic to extract the "where" from any model.

Reply:
Yep, I agree this is unnecessarily verbose, there may be a better way to do it